src/scss/core/components/_pagination.scss

```diff
@@ -9,6 +9,7 @@
 
   &-container {
     align-items: center;
-    margin: 1rem 0;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
   }
```

Comment:
I'd rather add proper classes to the template (`my-3 align-items-center`) instead of duplicating CSS that is already in Bootstrap. The theme should be as light as possible. Custom CSS should be added only to custom components that can't be styled directly with BS or in loops to limit the size of compiled php file